New

Bolbochan opened the pull request:
Планувалося що num.soap-змінить своє значення, але він не хоче. Тому було прийнято рішення записувати через console.log() до вирішення проблеми.

Bolbochan committed:
Added test version

Bolbochan committed:
added new version but it is not perfect

Andriy-tut commented:
Я трохи не розумію в чому суть проблеми. Можеш написати комент хоча б біля тої функції яку ти маєш на увазі

Bolbochan committed:
Update one.js

Рядок 37(Домашнє завдання має умову 'Учтите, что при добавлении покупки с уже существующим в списке продуктом, необходимо увеличивать количество в существующей покупке, а не добавлять новую.')
//Функція Two кожного разу підраховує зміну кількості продуктів, але якщо задати 'num.soap = num.soap + +indicate' число змінної залишається незмінним, хоча якщо виводити його через  'console.log(num.soap = num.soap + +indicate)' воно правильне. 
Для прикладу  за умови що ми Soap-2  а ми додали в список ще Soap-3 то 'num.soap = num.soap + +indicate' дасть нам все одно 2, але якщо ж цю функцію вивести зразу через 'console.log(num.soap = num.soap + +indicate)' то дає правильну кількість 5.
